style.css

```diff
@@ -76,14 +69,239 @@ h2 {
   color: #7f8cff;
 }
 
+/* Works section */
+.works-container,
+.multi-post-container {
+  display: grid;
+  grid-template-rows: auto;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
+
+.card-one {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+}
+
+.card-two {
+  margin-left: 25px;
+}
+
+.three-elements-row {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 75px;
+  padding: 20px 0;
+}
+
+.three-elements {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+}
+
+.tonic-title,
+.multi-post-title,
+.stories-title {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172d4d;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` etc. which makes your design `inconsistent` and looks unprofessional. Please remove the extra space and use it and have a look at the below-attached image and table. 

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 17-50-45](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/1d1fd44b-1cc0-495f-a071-ca4bdefe590e)

</td>
<td>

![Screenshot from 2023-06-10 17-50-38](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/5a5511db-ae46-4ca1-bcc1-8b0d52c3a7ec)

</td>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 17-51-24](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/75ddc093-d873-4e7f-a62a-f58383e97691)

</td>
<td>

![Screenshot from 2023-06-10 17-51-38](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/9c3a067d-f023-41f3-9ae4-48c90854b699)

</td>
</tr>
</table>


### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)